src/control/control.h

```diff
@@ -115,8 +115,14 @@ class ControlDoublePrivate : public QObject {
     // confirmed by setAndConfirm() or not. Note: Once connected, the CO value
     // itself is ONLY set by setAndConfirm() typically called in the connected
     // slot.
-    bool connectValueChangeRequest(const QObject* receiver,
-                                   const char* method, Qt::ConnectionType type);
+    template <typename Receiver, typename Slot>
+    bool connectValueChangeRequest(Receiver receiver,
```

Comment:
This way there might be less type savety.
Does the compiler complain if slot is not part of the receiver?

Reply:
Yes, it will not compile if the slot is not part of the receiver.